chat-with-tor/libp2p-demo-tutorial.md

```diff
@@ -0,0 +1,235 @@
+# Building a P2P app with go-libp2p
+
+Libp2p is a peer-to-peer networking library that allows developers to easily add p2p connectivity between their users. Setting it up and using it is simple and easy! In this brief tutorial, we'll create a simple peer-to-peer chat application with optional support for tor transports.
+
+To start, make sure you have Go installed and set up. Then install libp2p and some other dependencies we need with:
+
+```shell
+$ make deps
```

Comment:
Same, `go get -u ./...` should work. Also note that we have a convention of using `>` as the shell prefix for reasons I can't recall.